packages/algob/src/lib/files.ts

```diff
@@ -48,3 +50,27 @@ export function loadEncodedTxFromFile (fileName: string): Uint8Array | undefined
     throw e;
   }
 }
+
+/**
+ * Load .js, .ts files from /scripts (default) directory
+ * @param directory directory to load files from
+ * @param taskType task type (eg. test)
+ * @returns array of paths as string eg. ['scripts/file1.js', 'scripts/file2.js', ..]
+ */
+export function loadFilenames (directory: string, taskType?: string): string[] {
```

Comment:
how about moving this somewhere in `/internal`? atm, i don't think we need it in `/src/lib`.

Reply:
Hi @ratik21! Nice to have conversations with you.

I see in the built-in tasks we use functions inside folder src/lib. This is the reason did I move the function to src/lib.

Reply:
I think it will keep the loadFilenames function in this file. It more sense if I do refactor code in another pull request.

Reply:
The idea is that we use `src/internal` for things we don't want to expose to the user. 
Is a user going to import `loadFilenames` in his local project? If the function is going to be used only in the built-in tasks or other package functions, then it makes more sense to move it to `src/internal`.

Reply:
Thanks! I got it.

Reply:
I only refactor loadFilenames function. I see some other files want to refactor too. I think I will do it in another pull request.